openlibrary/plugins/openlibrary/js/FileValidation.js

```diff
@@ -0,0 +1,29 @@
+export default function Filevalidation (){
+ var fi = document.getElementById('coverFile'); 
+ if (fi.files.length > 0) {
+  for (var i = 0; i <= fi.files.length - 1; i++) {
+       var fsize = fi.files.item(i).size; 
+       var filname=fi.files.item(i).type;
+       var filtyp=filname.split('/')[0];
+       var file = (fsize / 1024); 
+       // The type of the file. 
+       if(!filtyp.startsWith('image')){
+         document.getElementById('imageUploadSize').innerHTML = 
+         '<b>Max Size Upload: 30MB.</b>';
```

Comment:
This needs i18n using the _ function which is only available in the template. Please move into the HTML and use JS to .show / .hide the message.

Reply:
https://github.com/internetarchive/openlibrary/wiki/Frontend-Guide#internationalization-i18n---for-programmers

Reply:
i18n is not available on the client side, so you'll need to do this in the template. Hide the element using display: none; and call `$('#imageUploadSize').show()` to display it. Make sure you hide it at the beginning of this function so subsequent upload attempts that are successful do not show a confusing error message.